generated/proto/placement/placement.proto

```diff
@@ -21,35 +21,35 @@ syntax = "proto3";
 
 package placement;
 
-message placement {
-  map<string, instance> instances = 1;
+message Placement {
+  map<string, Instance> instances = 1;
   uint32 replica_factor = 2;
   uint32 num_shards = 3;
   bool is_sharded = 4;
   int64 cutover_time = 5;
 }
 
-message instance {
+message Instance {
   string id = 1;
   string rack = 2;
   string zone = 3;
   uint32 weight = 4;
   string endpoint = 5;
-  repeated shard shards = 6;
+  repeated Shard shards = 6;
 }
 
-message shard {
+message Shard {
   uint32 id = 1;
-  shard_state state = 2;
+  ShardState state = 2;
   string source_id = 3;
 }
 
-enum shard_state {
-  Initializing = 0;
-  Available = 1;
-  Leaving = 2;
+enum ShardState {
+  INITIALIZING = 0;
```

Comment:
It's recommended to have 'UNKNOWN' as the first value in an enum. This will cause older clients to default to 'UNKNOWN' if they encounter new enum values. If we don't have active users of this type, could you make that change too.